app/scripts/modules/kubernetes/src/v2/manifest/delete/delete.html

```diff
@@ -0,0 +1,52 @@
+<div modal-page class="confirmation-modal">
+  <task-monitor monitor="ctrl.taskMonitor"></task-monitor>
+  <form role="form" name="deleteForm">
+    <modal-close dismiss="$dismiss()"></modal-close>
+    <div class="modal-header">
+      <h3>Delete {{ctrl.command.name}} in {{ctrl.command.location}}</h3>
```

Comment:
It would be nice to un-camelcase replicaSet here. You can use `{{ ... | robotToHuman }}`